go.mod

```diff
@@ -146,7 +146,7 @@ require (
 	github.com/mdlayher/netlink v1.7.2
 	github.com/microsoft/go-mssqldb v1.7.2 // replaced
 	github.com/microsoft/kiota-authentication-azure-go v1.0.2
-	github.com/microsoftgraph/msgraph-sdk-go v1.40.0
+	github.com/microsoftgraph/msgraph-sdk-go v1.45.0
```

Comment:
Rather large jump, but it looks like that's how they work.

https://github.com/microsoftgraph/msgraph-sdk-go/releases